src/path.h

```diff
@@ -62,7 +63,9 @@ struct get_path_result_t {
 get_path_result_t path_try_get_path(const wcstring &cmd, const environment_t &vars);
 
 /// Return all the paths that match the given command.
-wcstring_list_t path_get_paths(const wcstring &cmd, const environment_t &vars);
+std::vector<std::wstring> path_get_paths(const wcstring &cmd, const environment_t &vars);
+
+std::vector<std::string> path_get_paths_from_parser(const wcstring &cmd, const parser_t &parser);
```

Comment:
we should mark this wrapper somehow so we know not to port it to Rust. Usually we add an `_ffi` suffix

Reply:
I wish I knew why autocxx was OK with the wide version of this everywhere except on macOS.

Reply:
It wasn't happy on FreeBSD either: https://cirrus-ci.com/task/5127876329078784.

(my guess is compiler versions, but tbh given that this is *not* a huge performance penalty I'm just gonna take it in this case - it's much more pressing in others)